lib/guest_installation_and_configuration_base.pm

```diff
@@ -253,7 +254,8 @@ sub initialize_guest_params {
     $self->{$_} //= '' foreach (keys %guest_params);
     $self->{host_ipaddr} = get_required_var('SUT_IP');
     $self->{host_name} = script_output("hostname");
-    $self->{host_domain_name} = script_output("dnsdomainname");
+    # For SUTs with multiple interfaces, `dnsdomainname` sometimes does not work
+    $self->{host_domain_name} = script_output("dnsdomainname", proceed_on_failure => 1);
```

Comment:
Can I have a look on this machine ?

Reply:
I appericiate you are ready to help, but the SUT is in O3 VLAN. Only The authorized people(based on sshkeys)  can access it from a jump server.  A easier way I can think of is to access the SUT from my laptop when we are at office, Let's talk offline. but I think we'd better let the fix in firstly. then we can try to find the root cause and fix it in no hurry. The root cause I can think of is that the two interfaces are in different VLAN, which may confuse 'dnsdomainname` at times.

Reply:
I think you can try on your laptop when you in the office, because your laptop has connections on both ethx and wlanx. Just check the return value of dnsdomainname, because there is no domain name on your laptop.

Reply:
At the moment, I have no access to AMD ZEN3 machine which also has multiple interfaces. There is no such issue on it.